views/download.handlebars

```diff
@@ -2,13 +2,13 @@
   <script src="/download.js"></script>
   <div id="download-page-one">
     <div class="title">
-      <span id="dl-filename"
+      <span id="dl-file"
+            data-filename="{{filename}}"
+            data-size="{{sizeInBytes}}"
+            data-ttl="{{ttl}}"
             data-l10n-id="downloadFileName"
             data-l10n-args='{"filename": "{{filename}}"}'></span>
```

Comment:
Nice catch @pdehaan. We ought to be doing `data-l10n-args='{{JSON.stringify({filename})}}'`